TrackingAntennaV3/main/Accelerometer.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file   Magnetometer.hpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ * @brief  
+ * @see https://www.arduino.cc/en/Reference/Ethernet
+ */
+
+#ifndef ACCELEROMETER
+#define ACCELEROMETER
+
+#define COMPASS_ID 12345
```

Comment:
Why compass id? Accelorometers have nothing to do with a compass. Rename this to something else, like ACCELOROMETER_I2C_ID if the sensor communicates via i2c